app/Http/Controllers/UserListController.php

```diff
@@ -150,7 +150,7 @@ public function show($userid, $listid){
         ]);
     }
 
-    public function index($userid){
+    public function index($userid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->